spring-security-modules/spring-security-oauth2-testing/servlet-resource-server/pom.xml

```diff
@@ -54,4 +56,9 @@
             </plugin>
         </plugins>
     </build>
```

Comment:
Why did we add this?

Reply:
same reason mentioned above removing this

Reply:
Could we remove ?

<properties>
        <java.version>17</java.version>
    </properties>

Reply:
Its removed